src/blockchain/deposit_strategy/interface.py

```diff
@@ -1,17 +1,57 @@
+import logging
 from abc import ABC, abstractmethod
 
+from blockchain.deposit_strategy.gas_price_verifier import GasPriceCalculator, get_pending_base_fee
 from blockchain.typings import Web3
+from metrics.metrics import DEPOSITABLE_ETHER, POSSIBLE_DEPOSITS_AMOUNT
+from web3.types import Wei
+
+logger = logging.getLogger(__name__)
 
 
 class ModuleDepositStrategyInterface(ABC):
-    def __init__(self, w3: Web3, module_id: int):
-        self.w3 = w3
-        self.module_id = module_id
+    _DEPOSITABLE_KEYS_THRESHOLD = 0
```

Comment:
Let's invert the condition:
If you have at least TRESHOLD keys, you can deposit

Reply:
Also add small description since this value will be different for different strategies

Reply:
Done